data/input_files/run_serac_tests.sh

```diff
@@ -0,0 +1,105 @@
+#! /bin/bash
+# desc: runs all inputs on serac moves their summary.json files
+# to the tests integration directory
+# notes:
+#  - this is meant to run in ./serac_repo/data/input_files
+#  - SERAC_EXE can be found in a build directory under "bin"
+#  - you can run without sbatch with IS_PARALLEL=0
+#  - KEEP_OUTPUTFILE for debugging
+
+#SBATCH -J run_serac_tests
+#SBATCH -t 5:00:00
```

Comment:
Most machines wont let you run a 5 hour job during the day.  It will just sit in the queue or The Hammer will kill it.  Put this to like 2 hours.